practice-app/backend/routes.js

```diff
@@ -1,7 +1,14 @@
 const nearbyVendors = require('./controllers/nearestVendor');
+const covidNews = require('./controllers/covidNews');
 
 // Initialize the routes.
 module.exports.initialize = function (app) {
     // Redirect the GET /vendor/nearest request to the getNearestVendor function.
     app.get("/vendor/nearest", nearbyVendors.getNearestVendor);
-}
\ No newline at end of file
+}
+
+module.exports.initialize = function (app) {
+    // Redirect the GET /vendor/nearest request to the getNearestVendor function.
+    app.get("/covid/news", covidNews.getNews);
```

Comment:
This part shouldn't be inside a new function(module.export.initialize part). The problem that @egirgin and @berkayalkan should be solved if you move this inside the previous function(function starting at 5th line).